src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/api/impl/PartyApiServiceImpl.scala

```diff
@@ -236,25 +236,28 @@ class PartyApiServiceImpl(
 
   }
 
+  private def parseArrayParameter(param: String): List[String] =
+    if (param == "[]") List.empty
+    else param.parseCommaSeparated
+
```

Comment:
if we use the current approach for list parameters, this function would be required in several services.
We could think to add it to the library?

Reply:
I do agree.